gst/nnstreamer/tensor_typedef.h

```diff
@@ -155,6 +155,8 @@ typedef struct _GstTensorFilterProperties
   int output_configured; /**< TRUE if output tensor is configured. Use int instead of gboolean because this is refered by custom plugins. */
   GstTensorsInfo output_meta; /**< configured output tensor info */
 
+  int use_nnapi; /**< TRUE if tensor filter uses NNAPI as tenserflow lite backend. */
+
```

Comment:
DO NOT ADD subplugin specific data in this global data structure.

Reply:
use_nnapi is needed to notify the use of NNAPI to tensorflow lite. Could you guide me where is proper place instead of GstTensorFilterProperties?

Reply:
```int use_nnapi``` should be in a struct allocated at ```void *privateData;``` of tensor_filter.h
Any struct containing ```int use_nnapi``` MUST be inside tensorflow-lite subplugin code.
It should NEVER be exposed to elsewhere.